packages/core/type-definitions/combinator/combine.d.ts

```diff
@@ -1,10 +1,6 @@
 import { Stream } from '@most/types';
 
 export function combine<A, B, R>(fn: (a: A, b: B) => R, a: Stream<A>, b: Stream<B>): Stream<R>;
```

Comment:
Similarly for the TS type defs, need to add the partial application variants for `merge`, `combine`, and `zip`.